papers/ice_nucleation_2024/calibration.jl

```diff
@@ -58,18 +49,7 @@ function run_model(p, coefficients, IN_mode, FT, IC, end_sim)
         )
         kaolinite_calibrated = CP.create_toml_dict(FT; override_file)
         overwrite = CMP.Kaolinite(kaolinite_calibrated)
-
-        # run parcel with new coefficients
-        local params = parcel_params{FT}(
-            const_dt = const_dt,
-            w = w,
-            aerosol = overwrite,
-            heterogeneous = heterogeneous,
-            condensation_growth = condensation_growth,
-            deposition_growth = deposition_growth,
-            liq_size_distribution = liq_size_distribution,
-            ice_size_distribution = ice_size_distribution,
-        )
+        aerosol = overwrite
```

Comment:
We could just delete this line and have `aerosol = CMP.Kaolinite(kaolinite_calibrated)`? And the same for other two options in this file?